test/slow-down.test.js

```diff
@@ -0,0 +1,32 @@
+import { test } from 'tap'
```

Comment:
isn't this test taking a long time to run?

Reply:
No, i removed that one.